rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,476 @@
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["VADOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["VA006","VA013","VA017","VA029","VA016","VA041","VA022","VA019","VA003","VA011","VA031","VA026","VA004","VA023","VA015","VA021","VA032","VA027","VA040"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["VTDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["VT002","VT001"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WIDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["EKNW3","WAKW3","WI054","HAUW3","PAKW3","HSNW3","MNRW3"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WXforYou"]
+           - variable: MetaData/stationIdentification
+             is_in: ["H0269","H0313","H0300","H0080","H0346","H0164","H0005","H0177","H0041","H0498","H0338","H0062","H0244","H0203","H0237","H0030","H0322","H0560","H0140","H0447"]
+         - filter: AcceptList  # accept back selected stations
+           apply at iterations: 0,1
+           where:
+           - variable: MetaData/mesonetProvider
+             is_in: ["WYDOT"]
+           - variable: MetaData/stationIdentification
+             is_in: ["KBIT","KBRX","KHLD","KBFU","KCMS","KPER","KDRC"]
+
+         # Bounds Check
+         - filter: Bounds Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           minvalue: -130
+           maxvalue: 130
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     [ 5.0 ]
```

Comment:
I think we also have a gross error check with a threshold * 0.7 when QualityMarker = 3 for wind observations?